APCAppCore/APCAppCore/UI/TabBarControllers/Dashboard/APCGraphViewController.m

```diff
@@ -59,6 +60,8 @@ - (void)viewDidLoad {
         self.lineGraphView.hidden = YES;
     }
     
+    [[NSNotificationCenter defaultCenter]addObserver:self selector:@selector(reloadCharts:) name:APCUpdateChartsNotification object:nil];
```

Comment:
If keeping this, it should at least be observing the specific scoring object, shouldn't it?